frontend/public/components/resource-quota.jsx

```diff
@@ -453,10 +586,43 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
   },
 );
 
-export const ResourceQuotasDetailsPage = (props) => (
-  <DetailsPage
-    {...props}
-    menuActions={resourceQuotaMenuActions}
-    pages={[navFactory.details(Details), navFactory.editYaml()]}
-  />
+export const AppliedClusterResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
+  ({ namespace, flags, mock, showTitle }) => {
+    const { t } = useTranslation();
+    const resources = [
+      {
+        kind: referenceForModel(AppliedClusterResourceQuotaModel),
+        namespaced: true,
+        namespace,
+        optional: true,
+      },
+    ];
+
+    if (flagPending(flags[FLAGS.OPENSHIFT])) {
+      return <LoadingBox />;
+    }
+
+    return (
+      <MultiListPage
+        ListComponent={ResourceQuotasList}
```

Comment:
We want a specific list component for AppliedClusterResourceQuota IMO that has different table columns.

Reply:
I think the label selector, target project annotations, and creation date would be good (for both AppliedClusterRresourceQuota and ClusterResourceQuota).